consistent_web_main/main.go

```diff
@@ -334,9 +334,7 @@ func main() {
 		// Initialize for time.Now() + 60 seconds to allow for starting everything up
 		timestamp := time.Now().Add(60 * time.Second)
 		// If using Google Cloud, change this variable to include the IPs of the servers you have created
-		nodeList := []consistent_hash.ServerNode{
-			{IP: "localhost", Timestamp: timestamp, Replicas: 1}
-		}
+		nodeList := []consistent_hash.ServerNode{{IP: "localhost", Timestamp: timestamp, Replicas: 1}}
```

Comment:
have you tested this with multiple servers yet? Make sure you do that as well to ensure that it is working perfectly.